### DISCUSSION TITLE
semver RC Versions for Go Packages

### DISCUSSION THREAD

**Discussion by tametsi** [pocketbase/pocketbase#5897] [Q&A] [answered]
Hi,
I've just noticed by updating to the latest v0.23 rc that go interprets `v0.23.0-rc9` > `v.0.23.0-rc14`.

I assume this is due to the missing dot between the `rc` and the pre-release version (expected `v0.23.0-rc.14`) as somewhat expected by [semver](https://semver.org/#spec-item-9) 

> A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers

Or am I wrong with this assumption?

Obviously this cannot be changed for the already released versions but it might be worth having a look at this for the next rc series ;).

Great project btw, really enjoy working with it <3

**Comment by ganigeorgiev** [comment, marked as answer]
There is no strict requirements for this and technically the last part could be anything, but Yes, maybe next time an rc is published a dot separation may prevent this from happening (_although this still would depend on how Go pkg parses the semver strings and I haven't really tested with another package > rc10_).

Anyway, this shouldn't really matter (_feel free to elaborate if it is causing an actual issue but if you have troubles fetching the latest release you can always refer to the last commit from the `develop` branch_).
